test/lint/licenses.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh -eu
 
+# FIXME: Re-enable when https://github.com/canonical/lxd/issues/13048 is resolved.
+# Note: Skipping here because `make static-analysis` calls `run-parts` on all files in this directory.
+return 0
```

Comment:
Not sure `return` is valid here. `exit 0` maybe?

Reply:
So with `/bin/sh` that's permitted but under `bash` that fails:
```
$ return 0
bash: return: can only `return' from a function or sourced script
```

Reply:
Ah ok I'll change to `exit 0`.